ginga/canvas/types/astro.py

```diff
@@ -1274,6 +1275,11 @@ def draw(self, viewer):
             # and update all the text objects in self.objects
             update = True
 
+        cur_limits = viewer.get_limits()
+        if np.any(np.isclose(cur_limits, self._cur_limits)):
```

Comment:
I thought it's like this?
```suggestion
        if not np.isclose(cur_limits, self._cur_limits):
```

Reply:
@pllim, doesn't work, because `np.isclose` returns an array, so you have to wrap with the `np.any`. I think it works nicely and is understandable.